dask/dataframe/io/tests/test_parquet.py

```diff
@@ -125,14 +125,14 @@ def test_roundtrip_from_pandas():
 
 def test_categorical():
     with tmpdir() as tmp:
-        df = pd.DataFrame({'x': ['a', 'b', 'c'] * 100},
+        df = pd.DataFrame({'x': [u'a', u'b', u'c'] * 100},
```

Comment:
These might be better resolved by adding a `__futures__` import line

    from __future__ import absolute_import, division, print_function, unicode_literals